mainnet/CAKE/data.json

```
@@ -0,0 +1,10 @@
+{
+  "chainId": 143,
+  "address": "0xF59D81cd43f620E722E07f9Cb3f6E41B031017a3",
+  "name": "PancakeSwap Token",
+  "symbol": "CAKE",
```

Looks like that the on-chain symbol is `Cake`:
```suggestion
  "symbol": "Cake",
```
Please, also update the folder name to `Cake` to match the symbol.